sympy/stats/tests/test_stochastic_process.py

```diff
@@ -134,3 +136,55 @@ def test_ContinuousMarkovChain():
     CS1 = ContinuousMarkovChain('C', [0, 1, 2], TS1)
     A = CS1.generator_matrix
     assert CS1.transition_probabilities(A)(t) == exp(t*A)
+
+def test_BernoulliProcess():
+
+    B = BernoulliProcess("B", p=0.6, success=1, failure=0)
+    assert B.state_space == FiniteSet(0, 1)
+    assert B.index_set == S.Naturals0
+    assert B.success == 1
+    assert B.failure == 0
+
+    X = BernoulliProcess("X", p=0.6, success='H', failure='T')
+    assert X.state_space == FiniteSet('H', 'T')
+    H, T = symbols("H,T")
+    assert str(E(X[1]+X[2]*X[3])) == str(0.36*H**2 + 0.48*H*T + 0.6*H + 0.16*T**2 + 0.4*T)
```

Comment:
Don't use `str` in tests. If this is because of float comparisons then use a `Rational` for `p` instead of a `float`.